_posts/0000-01-02-Spinde24.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I would like to learn to surf, but if I will learn how to code it would still be okay